### PR TITLE
Add support for inline ConfigMaps in StackSet CRD

### PR DESCRIPTION
**Pull request by linki**
Update StackSet CRD for inline ConfigMaps. This is just an early version to update CRD in stackset-controller's e2e cluster to be able to run the test cases.

### CODE REVIEW

**Comment by linki** [comment]
👍

**Comment by linki** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

File: cluster/manifests/stackset-controller/01-stack-crd.yaml [hunk 1]
```diff
@@ -357,6 +357,20 @@ spec:
...
+                        data:
+                          additionalProperties:
+                            type: string
+                          description: Data of the ConfigMap
+                          type: object
+                        name:
+                          description: Name of the ConfigMap
+                          type: string
+                      type: object
+{{ end }}
```
Comment (AlexanderYastrebov) [inline cluster/manifests/stackset-controller/01-stack-crd.yaml:373]: Looks like these were not added to the origin stack CRD https://github.com/zalando-incubator/stackset-controller/blob/master/docs/stack_crd.yaml

Comment (AlexanderYastrebov) [reply cluster/manifests/stackset-controller/01-stack-crd.yaml:373]: Noticed while working on https://github.com/zalando-incubator/kubernetes-on-aws/pull/7297

Comment (linki) [reply cluster/manifests/stackset-controller/01-stack-crd.yaml:373]: Thanks. Yes, this wasn't merged 🤦  I created a separate PR to fix it: https://github.com/zalando-incubator/stackset-controller/pull/621

Comment (linki) [reply cluster/manifests/stackset-controller/01-stack-crd.yaml:373]: We merged it to master now. It's slightly different than here (code comments) but the structure is the same. Let's update it in one of the subsequent PRs that modify the spec.

Comment (AlexanderYastrebov) [reply cluster/manifests/stackset-controller/01-stack-crd.yaml:373]: Thank you, I will update that within https://github.com/zalando-incubator/kubernetes-on-aws/pull/7297

**Pull request merged**